buildenv/jenkins/common/build.groovy

```diff
@@ -326,8 +326,62 @@ def archive_sdk() {
     }
 }
 
+// When error occurs in java, there will be an output in the log IEATDUMP success for DSN='JENKINS.JVM.JENKIN*.*.*.X&DS'
+// The X&DS is a macro that is passed to the IEATDUMP service on z/OS
+// The dump will be split into multiple files by the marco if the address space is large
+// Each dataset is named in a similar way but there is a distinct suffix to specify the order 
+// For example, JVM.*.*.*.001, JVM.*.*.*.002, JVM.*.*.*.003
+// The function fetchFile moves each dataset into the current directory of the machine, and appends the data to the core.xxx.dmp at the same time
+// The file core.xxx.dmp will be uploaded to resultstore for triage
+
```

Comment:
Pls remove this blank line

Reply:
> Pls remove this blank line

Thanks for your help! The comments were fixed. Thanks!